hail/python/hailtop/gear/auth/exceptions.py

```diff
@@ -0,0 +1,7 @@
+class NoTokenFileFoundError(Exception):
+    def __init__(self, tokens_file):
+        self.tokens_file = tokens_file
+        self.message = (f'Cannot authenticate because no tokens file was found '
+                        f'at {tokens_file}. Execute `hailctl auth login` to '
+                        f'obtain tokens.')
```

Comment:
Couple remarks:
 - Do we want thrown exceptions to be error messages in hailctl?  I'd argue no.
 - This probably needs to say what namespace the token needs to be obtained for.  If it's default, this should be the error message and we shouldn't mention the namespace.  For non-default, it is a developer error message, and the namespace should be present (and depending on the current configuration, `hailctl auth login` might not be sufficient.)